packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -113,6 +113,19 @@ export class GarbageCollector implements IGarbageCollector {
 	private readonly summaryStateTracker: GCSummaryStateTracker;
 	private readonly telemetryTracker: GCTelemetryTracker;
 
+	/** If false, loading or using a Tombstoned object should merely log, not fail */
+	public get gcTombstoneEnforcementAllowed(): boolean {
+		return this.configs.gcTombstoneEnforcementAllowed;
+	}
+	/** If true, throw an error when a tombstone data store is retrieved */
+	public get gcThrowOnTombstoneLoad(): boolean {
+		return this.configs.gcThrowOnTombstoneLoad;
+	}
+	/** If true, throw an error when a tombstone data store is used. */
```

Comment:
```suggestion
	/** If true, throw an error when a tombstone data store is used (changed or revived). */
```

Right?

Reply:
Not really. We don't throw on revived today.